ci/nightly/pipeline.yml

```diff
@@ -590,6 +591,17 @@ steps:
       - ./ci/plugins/mzcompose:
           composition: cloud-canary
 
+  - id: output-consistency
+    label: "Output consistency"
+    timeout_in_minutes: 58
+    agents:
+      queue: linux-x86_64
+    artifact_paths: junit_*.xml
+    plugins:
+      - ./ci/plugins/mzcompose:
+          composition: output-consistency
+          args: ["--seed=$BUILDKITE_JOB_ID", "--max-runtime-in-sec=180"]
```

Comment:
Why did you decide on a 3-minute run in Nightly? I would say 10-30 min is more in line with the rest of the fuzzing workloads that are in Nightly (unless you consider the marginal utility from longer runs to be very minimal).

Reply:
✅ Changed to 20 mins. I was surprised how fast it is (at least compared to what I had expected 😀). However, it is way faster locally than in the CI.